lib/miq_expression/tag.rb

```diff
@@ -19,11 +20,12 @@ def self.parse(field)
 
   def initialize(model, associations, column, managed = true)
     super(model, associations, column)
-    @namespace = "/#{managed ? MANAGED_NAMESPACE : USER_NAMESPACE}/#{column}"
+    @namespace = managed ? MANAGED_NAMESPACE : USER_NAMESPACE
+    @path = "/#{namespace}/#{column}"
   end
 
   def contains(value)
-    ids = model.find_tagged_with(:any => value, :ns => namespace).pluck(:id)
+    ids = model.find_tagged_with(:any => value, :ns => path).pluck(:id)
```

Comment:
I think as this is a bug I would do the simplest thing to get it to :green_heart: , perhaps let's look at naming in a follow up PR? :D